include/Track.h

```diff
@@ -192,7 +192,12 @@ class TrackContentObjectView : public selectableObject, public ModelView
 
 // theming qproperties
 	Q_PROPERTY( QColor fgColor READ fgColor WRITE setFgColor )
+	Q_PROPERTY( QColor fgMutedColor READ fgMutedColor WRITE setFgMutedColor )
+	Q_PROPERTY( QColor mutedColor READ mutedColor WRITE setMutedColor )
+	Q_PROPERTY( QColor selectedColor READ selectedColor WRITE setSelectedColor )
 	Q_PROPERTY( QColor textColor READ textColor WRITE setTextColor )
+	Q_PROPERTY( QColor textShadowColor READ textShadowColor WRITE setTextShadowColor )
+	Q_PROPERTY( bool gradient READ gradient WRITE setGradient )
```

Comment:
? The colors get manualy converted into a linear gradient. If we set its alpha to 0 you would get a glitched pattern: 
![screenshot from 2016-02-15 23 36 50](https://cloud.githubusercontent.com/assets/6282045/13184791/2a77beb6-d73e-11e5-84db-6ea364b689ca.png)
Since there isn't anything drawn underneath it.